Jenkinsfile.yml

```diff
@@ -35,6 +35,15 @@ node('ccode'){
       '''
     }
 
+    //CSDK binaries are currently built without MODULES flag and hence don't support the ServiceInfo transfers.
+    //Enabling the flag temporarily for the smoke-test in cDevice_Build.sh. Stage will be removed once the script is modified.
+    stage('Add MODULES flag to build script'){
+      sh '''
+        cd $WORKSPACE/client-sdk
+        sed -i 's/-DPK_ENC=ecdsa/-DPK_ENC=ecdsa -DMODULES=true/g' cDevice_Build.sh
```

Comment:
There are 8 instances of `-DPK_ENC=ecdsa` in the script, Do we need to update all the 8 instances to `-DPK_ENC=ecdsa -DMODULES=true`?